specs/src/celestia-da.md

```diff
@@ -0,0 +1,78 @@
+# Celestia DA
+
+## Abstract
+
+This package implements the generic DA interface defined in [go-da] for Celestia.
+
+## Details
+
+The generic DA interface defines how DA implementations can submit, retrieve and validate blobs.
+
+The Celestia implementation is a wrapper around [celestia-openrpc] which
```

Comment:
Does not wrap celestia openrpc so this has to be adapted

Reply:
Done.